lib/SPIRV/OCLUtil.cpp

```diff
@@ -421,6 +447,6 @@ mutateFunctionOCL(Function *F,
 void
 llvm::MangleOpenCLBuiltin(const std::string &UniqName,
     ArrayRef<Type*> ArgTypes, std::string &MangledName) {
-  OCLUtil::OCLBuiltinFuncMangleInfo BtnInfo;
+  OCLUtil::OCLBuiltinFuncMangleInfo BtnInfo(nullptr);
```

Comment:
For the three device execution built-ins it is consistent. It uses another way to get built-ins mangled.